File: website/cgi-bin/envir.php
#!/usr/bin/env php
<?php

// Output the HTTP headers
header("Content-Type: text/html");

// Generate the HTML content
echo "<!DOCTYPE html>";
echo "<html>";
echo "<head><title>Environment Variables</title></head>";
echo "<body>";
echo "<h1>Environment Variables</h1>";
echo "<ul>";

// Iterate over the environment variables and print them
foreach ($_SERVER as $key => $value) {
    echo "<li><strong>$key:</strong> ";
    if (is_array($value)) {
        echo "<pre>" . print_r($value, true) . "</pre>";
    } else {
        echo htmlspecialchars($value);
    }
    echo "</li>";
}

echo "</ul>";
echo "</body>";
echo "</html>";
?>

File: www/website/cgi-bin/envir.pl

use strict;
use warnings;

print "Content-Type: text/html\n\n";
print "<!DOCTYPE html>\n";
print "<html>\n";
print "<head><title>Environment Variables</title></head>\n";
print "<body>\n";
print "<h1>Environment Variables</h1>\n";
print "<ul>\n";

foreach my $key (sort keys %ENV) {
    print "<li><strong>$key:</strong> $ENV{$key}</li>\n";
}

print "</ul>\n";
print "</body>\n";
print "</html>\n";